### PR TITLE
Feature interface standard lib

### PR DESCRIPTION
**Pull request by Kaaserne**
Added new files. Each file is responsible for their own calculations

### CODE REVIEW

File: code/headers/mlx_parameters.hpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+         */
+        static constexpr uint8_t TA0 = 25;
+        /**
+         * Emissivity compensation: We assume Emissivity = 1.
+         * Emissivity coefficient is user defined and it is not stored in the
+         * device EEPROM)
+         */
+        float emissivity;
+        // Kvdd, required for calculations
+        int Kvdd;
```
Comment (florianhumblot) [inline code/headers/mlx_parameters.hpp:21]: Please either explain what the value is, or reference the datasheet with a page number

Comment (florianhumblot) [reply code/headers/mlx_parameters.hpp:21]: Same goes for the other variables.

File: code/src/alpha_comp_extractor.cpp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                                                   int col)
+        : mlx_extractor_c(bus), row(row), col(col) {
+    }
+
+    void alpha_comp_extractor_c::extract(mlx_parameters_s &params) {
+        int data;
+
+        data = bus.read_register(registers::EE_ALPHA_ACC_SCALE);
+        const int alpha_scale =
+            data_extractor_s::extract_data(data, 0xF000, 12) + 30;
```
Comment (florianhumblot) [inline code/src/alpha_comp_extractor.cpp:15]: Please give a clear name to your magic numbers or explain them in a comment.

Comment (florianhumblot) [reply code/src/alpha_comp_extractor.cpp:15]: I'll only leave this comment here, but the same goes for your other uses of magic numbers.

**Review by florianhumblot — CHANGES_REQUESTED** [review]
Looks pretty good. A few small changes are needed though.

File: code/headers/mlx_parameters.hpp (hunk 1 above)
Comment (Kaaserne) [reply code/headers/mlx_parameters.hpp:21]: Please see the pipeline `mlx90640_processor_c::get_temperature_pixel`. Each object is (approximately) responsible for its own calculation. Above the creation of each object in `mlx90640_processor_c::get_temperature_pixel`, there is a reference to a section in the datasheet, in which these numbers come back.

File: code/src/alpha_comp_extractor.cpp (hunk 2 above)
Comment (Kaaserne) [reply code/src/alpha_comp_extractor.cpp:15]: > 
> 
> Please see the pipeline `mlx90640_processor_c::get_temperature_pixel`. Each object is (approximately) responsible for its own calculation. Above the creation of each object in `mlx90640_processor_c::get_temperature_pixel`, there is a reference to a section in the datasheet, in which these numbers come back.

**Review by florianhumblot — APPROVED** [review]

**Pull request merged**